videomaker/inc/patterns/header-description-and-grid-of-projects.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Header, Description and Grid of Projects
+ *
+ * @package Videomaker
+ */
+
+return array(
+	'title'      => __( 'Header, Description and Grid of Projects', 'videomaker' ),
+	'categories' => array( 'videomaker' ),
+	'blockTypes' => array( 'core/template-part/header' ),
+	'content'    => '<!-- wp:columns {"verticalAlignment":"top","align":"wide"} -->
+		<div class="wp-block-columns alignwide are-vertically-aligned-top"><!-- wp:column {"verticalAlignment":"top","width":"35%"} -->
+		<div class="wp-block-column is-vertically-aligned-top" style="flex-basis:35%">
+
+		<!-- wp:site-title /-->
+
+		<!-- wp:heading {"level":3,"style":{"typography":{"fontWeight":"300"}}} -->
```

Comment:
@beafialho Should these headings match H3s in the theme? What should the default weight for a heading be?

Reply:
@scruffian you mean the description "Gary is Creative Director of Hano..."? That should be a Large Paragraph and the default weight for headings should be Bold

<img width="885" alt="Captura de ecrã 2021-09-24, às 17 28 50" src="https://user-images.githubusercontent.com/65220155/134709506-8893d92f-553a-4404-9997-1794ad1b3c7f.png">